clang/test/Driver/sycl-offload.cpp

```diff
@@ -104,3 +104,7 @@
 // CHECK_FSYCL_FP64_CONV_EMU_WIN-NOT: clang{{.*}} "-cc1" "-triple x86_64-unknown-linux-gnu" {{.*}} "-fsycl-fp64-conv-emu"
 // CHECK_FSYCL_FP64_CONV_EMU_WIN-DAG: clang{{.*}} "-cc1" "-triple" "spir64_gen{{.*}}" "-fsycl-fp64-conv-emu"
 // CHECK_FSYCL_FP64_CONV_EMU_WIN-DAG: clang-offload-packager{{.*}} "--image=file={{.*}}.bc,triple=spir64_gen-unknown-unknown,arch=,kind=sycl,compile-opts={{.*}}-options -ze-fp64-gen-conv-emu{{.*}}"
+
+/// Compilation checks to make sure an early empty -fsycl-targets does not
+/// crash.
+// RUN: %clangxx -### -fsycl -fsycl-targets= -fsycl-targets=spir64 %s
```

Comment:
Is this supposed to be just a RUN command?
If yes, can we instead check if `spir64` target flow is achieved?

Reply:
It is just a check to see if the command returns an error.  I will update to check for the device compilation.